dockerfiles/Dockerfile.base

```diff
@@ -0,0 +1,5 @@
+FROM python:3.6.3-alpine3.6
```

Comment:
Let's call this Dockerfile.alpine. And merge this with Dockerfile.config. They don't need to be separate files.